steamship.json

```diff
@@ -31,6 +31,41 @@
       "type": "string",
       "description": "The OpenAI model to use.  Can be a pre-existing fine-tuned model.",
       "default": "text-davinci-002"
+    },
+    "top_p": {
+      "type": "number",
+      "description": "Controls the nucleus sampling, where the model considers the results of the tokens with top_p probability mass. Values between 0-1.",
+      "default": 1
+    },
+    "n_completions": {
+      "type": "number",
+      "description": "How many completions to generate for each prompt.",
+      "default": 1
+    },
+    "echo": {
+      "type": "boolean",
+      "description": "Echo back the prompt in addition to the completion",
+      "default": false
+    },
+    "stop": {
+      "type": "string",
```

Comment:
with this being a `string` but the config being List[str], should we clarify how the list should be passed in somewhere?

Reply:
Presumably in the description of the param